app/Controllers/userController.php

```diff
@@ -59,13 +59,23 @@ public static function updateUser($user, $email, $passwordPlain, $apiPasswordPla
 			$apiPasswordHash = self::hashPassword($apiPasswordPlain);
 			$userConfig->apiPasswordHash = $apiPasswordHash;
 
-			@mkdir(DATA_PATH . '/fever/', 0770, true);
+			$feverPath = DATA_PATH . '/fever/';
+
+			if (!file_exists($feverPath)) {
+				@mkdir($feverPath, 0770, true);
+			}
+
+			if (!is_writable($feverPath)) {
+				Minz_Log::error("Could not save fever API credentials. '{$feverPath}' does not have write access.");
+				return false;
```

Comment:
Should we make that a hard error and give up saving the rest (as we do now), or a warning, which will just prevent Fever API from working but not the rest?

Reply:
Either way is fine as long as we know what is happening.

Reply:
I think logging a warning and continue would be better

Reply:
OK, I'll modify the logic later.

Reply:
I've modified the logic but I am not convinced it's a good thing :/
It feels odd.

Reply:
@Alkarex I was using it locally without the error and it's really odd because it says that the user has been updated but then there are some errors you are not aware of since they are buried in the logs. I think it's better to have the hard error. What do you think?